CONTRIBUTING.md

```diff
@@ -152,6 +152,21 @@ rustup override set master
 cargo test
 ```
 
+After fixing the build failure on this repository, we can submit a pull request
+to [`rust-lang/rust`] to fix the toolstate.
+
+To submit a pull request, you should follow these steps:
+
+```
```

Comment:
```suggestion
```bash
```

Reply:
Same here?

Reply:
I'd rather add it, since the rust code blocks have this annotation. Maybe also add it above

Reply:
Added the annotations to this and the above snippet.